scipy/signal/_delegators.py

```diff
@@ -518,7 +522,7 @@ def symiirorder1_signature(signal, c0, z1, precision=-1.0):
 
 
 def symiirorder2_signature(input, r, omega, precision=-1.0):
-    return array_namespace(input)
+    return array_namespace(input, _skip_if_float(r), _skip_if_float(omega))
```

Comment:
`r` and `omega` are allowed to be `xp` "scalars", but not `precision`.  It this just a judgement call, or is there some other reason to treat `precision` differently?

Same remarks applies to `symiirorder1` arguments `c0` and `z1`.

Reply:
The only reason is that some tests have r and/or omega as numpy scalars. On other backends these become zero-dim arrays, and this is a workaround.